Dynavity/Dynavity/view/SideMenuView.swift

```diff
@@ -26,6 +29,15 @@ struct SideMenuView: View {
                 Text(canvasName)
                     .bold()
             }
+            if let onlineCanvas = canvasViewModel.canvas as? OnlineCanvas {
+                SideMenuContentView(label: "Shareable ID") {
+                    Text(onlineCanvas.shareableId)
+                }
+            } else {
+                Button("Publish") {
+                    sideMenuViewModel.publishCanvas(canvasViewModel: canvasViewModel)
```

Comment:
My original intention was to allow `SideMenuViewModel` to make any additional actions if necessary instead of interacting directly with another view-model, which can create coupling across different view/view-models.

But the logic turns out to be quite trivial and I see that `GraphViewModel` is being directly accessed too, so I guess publishing directly should be ok.